website/src/components/SearchPage/fields/AutoCompleteField.tsx

```diff
@@ -60,101 +59,99 @@ export const AutoCompleteField = ({
     }, [options, query]);
 
     return (
-        <Combobox
-            immediate
-            value={fieldValue}
-            onChange={(value) => setSomeFieldValues([field.name, value ?? ''])}
-            disabled={!isClient}
-        >
-            <div className='relative'>
-                <ComboboxInput
-                    displayValue={(value: string) => value}
-                    onChange={(event) => setQuery(event.target.value)}
-                    onFocus={load}
-                    placeholder={field.displayName ?? field.name}
-                    as={CustomInput}
-                    disabled={!isClient}
```

Comment:
IIUC this disabled will no longer be present due to the new component just setting disabled on the first child - but it's ok, no need to disable combobox input if the parent combobox is disabled

Reply:
yeah I guess nothing would stop us from nesting the new component but let's see if that's needed?